src/main/kotlin/no/nav/syfo/identhendelse/IdenthendelseService.kt

```diff
@@ -12,7 +11,7 @@ import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 class IdenthendelseService(
-    private val database: DatabaseInterface,
+    private val aktivitetskravService: AktivitetskravService,
```

Comment:
Hmm, jeg synes det gir mer mening å bare dra inn repositoriet her i stedet for å flyttet funksjonaliteten til `aktivitetskravService`. Oppdatering av personident er veldig spesifikt for identhendelse, og ikke så relevant for domenelogikken til Aktivitetskrav, så jeg synes kanskje ikke den funksjonen hører hjemme i `aktivitetskravService`. Tenker det er en funksjon vi helst vil at ikke skal være så tilgjengelig.

Reply:
Kan være enig i det slik det ser ut nå 😊
Egentlig tenkte jeg at alt inni `handle` funksjonen burde vært en funksjon i `AktivitetskravService` (noe slik kanskje `updateOutdatedPersonIdenter(…)`).  `IdentHendelseService` beskriver mer hva som skjer enn domenet.
Dette er jo egentlig en event som utelukkende påvirker `Aktivitetskrav`'et som domene og ressurs. Den har til og med noe kafka logging og metrics som skjer inni funksjonen sin.
Tenker at en fremtidig refaktorering kunne vært å kalle `updateOutdatedPersonIdenter(…)` direkte fra `KafkaIdenthendelseService` (`IdenthendelseConsumer` er kanskje bedre?) 

👆 Vi tar det en annen gang. Ville bare dele litt tanker jeg gjorde meg imens jeg refaktorerte 😊

Reply:
Enig i at logikken rundt identhendelse ikke nødvendigvis er helt prima, så det går det an å se mer på ja 😄